Wrapping/Generators/Python/Tests/CMakeLists.txt

```diff
@@ -49,70 +37,13 @@ if(ITK_WRAP_unsigned_char AND WRAP_2)
         DATA{${WrapITK_SOURCE_DIR}/images/cthead1.png}
         5
       )
-    if(_have_numpy_return_code EQUAL 0)
-      itk_python_add_test(NAME PythonExtrasTest
-        COMMAND extras.py
-          DATA{${WrapITK_SOURCE_DIR}/images/cthead1.png} ${ITK_TEST_OUTPUT_DIR}/out.png ${ITK_TEST_OUTPUT_DIR}
-        )
-    endif()
+    itk_python_add_test(NAME PythonExtrasTest
+      COMMAND extras.py
+        DATA{${WrapITK_SOURCE_DIR}/images/cthead1.png} ${ITK_TEST_OUTPUT_DIR}/out.png ${ITK_TEST_OUTPUT_DIR}
+      )
   endif()
-
-#   itk_python_add_test(NAME PythonStrelFromImageGrayscaleDilateImageFilter
-#     StrelFromImageGrayscaleDilateImageFilter.py
-#     DATA{${WrapITK_SOURCE_DIR}/images/cthead1.png}
-#     StrelFromImageGrayscaleDilateImageFilter.png
-#     DATA{${WrapITK_SOURCE_DIR}/images/StrelFromImage.png}
-#     --compare StrelFromImageGrayscaleDilateImageFilter.png DATA{${WrapITK_SOURCE_DIR}/images/StrelFromImageGrayscaleDilateImageFilter.png}
-#   )
-
-#   itk_python_add_test(NAME PythonFlatStructuringElementBall
-#     FlatStructuringElement.py
```

Comment:
Instead of simply removing this test, could you remove the part that does not work anymore (`GetImageUC`) and move the test to where it should be (`Modules/Filtering/MathematicalMorphology/test/`)?